packages/chart/src/components/SparkLine.js

```diff
@@ -135,6 +139,9 @@ export default function SparkLine({ width: parentWidth, height: parentHeight })
                         style={{ fill: colorScale ? colorScale(config.runtime.seriesLabels ? config.runtime.seriesLabels[seriesKey] : seriesKey) : '#000' }}
                         data-tip={tooltip}
                         data-for={`cdc-open-viz-tooltip-${config.runtime.uniqueId}`}
+                        defined={(item, i) => {
```

Comment:
I'm not sure that circles have a defined prop:
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/circle

When its a visx linepath you can use that trick. See ln 114 on LineChart.tsx to see how we're handling text and circles.

Reply:
removing the defined.